src/core/server/index.ts

```diff
@@ -210,12 +212,18 @@ export {
   SavedObjectsMigrationVersion,
 } from './types';
 
-export { LegacyServiceSetupDeps, LegacyServiceStartDeps } from './legacy';
+export {
+  LegacyServiceSetupDeps,
+  LegacyServiceStartDeps,
+  LegacyServiceDiscoverPlugins,
+} from './legacy';
```

Comment:
Don't think this is necessary? Plugins shouldn't need this for anything right?

Reply:
I needed the type in rendering.